### PR TITLE
docs(executors): add missing docstrings

### PR DESCRIPTION
**Pull request by Yongxuanzhang**

### CODE REVIEW

File: jina/executors/compound.py [hunk 1]
```diff
@@ -266,6 +266,14 @@ def components(self, comps: Callable[[], List]) -> None:
 
     @staticmethod
     def get_component_workspace_from_compound_workspace(compound_workspace: str, compound_name: str, pea_id: int) -> str:
+        """
+        Get the name of workspace.
+
+        :param compound_workspace: Workspace of the compound executor.
+        :param compound_name: Name of the compound executor.
+        :param pea_id: Id of the pea.
+        :return:
```
Comment (theUnkownName) [inline jina/executors/compound.py:275]: Can you add what is returned?

File: jina/executors/evaluators/embedding/__init__.py [hunk 2]
```diff
@@ -20,6 +20,12 @@ def evaluate(self, actual: 'np.array', desired: 'np.array', *args, **kwargs) ->
 
 
 def expand_vector(vec):
+    """
+
+
+    :param vec:
```
Comment (theUnkownName) [inline jina/executors/evaluators/embedding/__init__.py:26]: Also can you add more details here?

File: jina/executors/indexers/vector.py [hunk 3]
```diff
@@ -363,6 +364,12 @@ def query(self, vectors: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np
         return indices, dist
 
     def build_advanced_index(self, vecs: 'np.ndarray') -> 'np.ndarray':
+        """
+        Build advanced index structure based on in-memory numpy ndarray, e.g. graph, tree, etc.
+
+        :param vecs: the raw numpy ndarray
+        :return:
```
Comment (theUnkownName) [inline jina/executors/indexers/vector.py:371]: return details

**Review by theUnkownName — CHANGES_REQUESTED** [review]

File: jina/executors/indexers/vector.py (hunk 3 above)
Comment (Yongxuanzhang) [reply jina/executors/indexers/vector.py:371]: This method seems not implemented. I cannot how it actually works.

File: jina/executors/evaluators/embedding/__init__.py (hunk 2 above)
Comment (Yongxuanzhang) [reply jina/executors/evaluators/embedding/__init__.py]: Done

File: jina/executors/compound.py (hunk 1 above)
Comment (Yongxuanzhang) [reply jina/executors/compound.py]: Done

**Pull request merged**